### PR TITLE
Add panic bunker toggle to admin menu

### PR DESCRIPTION
**Pull request by Morb0**
## About the PR

A very useful feature that admins forget about

**Media**

image

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

### CODE REVIEW

File: Content.Client/Administration/UI/Tabs/ServerTab.xaml.cs [hunk 1]
```diff
@@ -18,6 +18,7 @@ public ServerTab()
 
             _config.OnValueChanged(CCVars.OocEnabled, OocEnabledChanged, true);
             _config.OnValueChanged(CCVars.LoocEnabled, LoocEnabledChanged, true);
+            _config.OnValueChanged(CCVars.PanicBunkerEnabled, BunkerEnabledChanged, true);
```
Comment (metalgearsloth) [inline Content.Client/Administration/UI/Tabs/ServerTab.xaml.cs:21]: You need to dispose this.

File: Content.Server/Administration/Commands/PanicBunkerCommand.cs [hunk 2]
```diff
@@ -12,21 +12,32 @@ public sealed class PanicBunkerCommand : IConsoleCommand
 
     public string Command => "panicbunker";
     public string Description => "Enables or disables the panic bunker functionality.";
-    public string Help => "panicbunker <enabled>";
+    public string Help => "panicbunker";
     public void Execute(IConsoleShell shell, string argStr, string[] args)
     {
-        if (args.Length != 1)
+        var cfg = IoCManager.Resolve<IConfigurationManager>();
```
Comment (metalgearsloth) [inline Content.Server/Administration/Commands/PanicBunkerCommand.cs:18]: This is already a dependency on the command.

File: Resources/Locale/en-US/administration/commands/panicbunker.ftl [hunk 3]
```diff
@@ -0,0 +1,2 @@
+panicbunker-command-enabled = Panic nubker has been enabled.
+panicbunker-command-disabled = Panic nubker has been disabled.
```
Comment (metalgearsloth) [inline Resources/Locale/en-US/administration/commands/panicbunker.ftl:2]: nubker

File: Content.Client/Administration/UI/Tabs/ServerTab.xaml [hunk 4]
```diff
@@ -9,5 +9,6 @@
         <cc:CommandButton Command="shutdown" Text="{Loc server-shutdown}" />
         <cc:CommandButton Name="SetOocButton" Command="setooc" Text="{Loc server-ooc-toggle}" ToggleMode="True" />
         <cc:CommandButton Name="SetLoocButton" Command="setlooc" Text="{Loc server-looc-toggle}" ToggleMode="True" />
+        <cc:CommandButton Name="SetPanicbunkerButton" Command="panicbunker" Text="{Loc server-panicbunker-toggle}" ToggleMode="True" />
```
Comment (metalgearsloth) [inline Content.Client/Administration/UI/Tabs/ServerTab.xaml:12]: This being enabled or not should be based on whether they have the perms to change it.

**Review by metalgearsloth — CHANGES_REQUESTED** [review]

File: Content.Client/Administration/UI/Tabs/ServerTab.xaml (hunk 4 above)
Comment (Morb0) [reply Content.Client/Administration/UI/Tabs/ServerTab.xaml:12]: It's in `Server` tab so there all commands are `AdminFlags.Server`

**Comment by Morb0** [comment]
I also removed the `Restart` button because it literally does nothing now.

File: Content.Client/Administration/UI/Tabs/ServerTab.xaml (hunk 4 above)
Comment (mirrorcult) [reply Content.Client/Administration/UI/Tabs/ServerTab.xaml:11]: commandbuttons already do this, this is fine:
image

**Review by mirrorcult — APPROVED** [review]

**Pull request merged**